apps/frontend/src/pages/profile/libs/components/edit-user-form/styles.module.css

```diff
@@ -2,18 +2,17 @@
 	display: flex;
 	flex-direction: column;
 	gap: 24px;
-	width: 100%;
+	align-items: flex-start;
 }
 
 .inputs-wrapper {
 	display: flex;
 	flex-direction: column;
 	gap: 16px;
 	width: 100%;
-	max-width: 500px;
+	min-width: 500px;
```

Comment:
I think we should keep max-width here, the problem why it's not full width is that one of the containers lacks width 100%